src/i18n/pt-BR/nav.ts

```diff
@@ -17,6 +17,7 @@ export default NavDictionary({
 	'guides/configuring-astro': 'Configurando Astro',
 	'core-concepts/framework-components': 'Frameworks de UI',
 	'guides/imports': 'Assets Estáticos',
+	'guides/aliases': 'Atalhos',
```

Comment:
Just to maintain the same pattern as the original docs, we can add "de Importação"
```suggestion
	'guides/aliases': 'Atalhos de Importação',
```